src/prefect/server/models/events.py

```diff
@@ -389,3 +389,49 @@ async def work_queue_status_event(
         related=related_work_pool_info,
         id=uuid4(),
     )
+
+
+async def work_pool_status_event(
+    event_id: UUID,
+    occurred: pendulum.DateTime,
+    pre_update_work_pool: Optional["ORMWorkPool"],
+    work_pool: "ORMWorkPool",
+):
+    assert work_pool.status
+
+    return Event(
+        id=event_id,
+        occurred=occurred,
+        # TODO: this should be
+        # event=f"prefect.work-pool.{work_pool.status.in_kebab_case()}",
+        event=f"prefect.work-pool.{work_pool.status.value.lower()}",
```

Comment:
Does this still need to be addressed?

Reply:
It will, but we'll have to do this in both Prefect Cloud and Prefect in tandem.  I've got a GH issue draft covering this.